### ISSUE TITLE
Misleading output in configure

### ISSUE THREAD

**Issue by svgmuc** [stevegrubb/libcap-ng#41]
```
--- configure.ac.orig   2023-04-27 22:18:57.659402571 -0400
+++ configure.ac        2023-04-27 22:19:07.173321549 -0400
@@ -60,7 +60,7 @@
        [AC_SEARCH_LIBS(pthread_atfork, pthread)],
        [AC_MSG_WARN(pthread.h not found, disabling pthread_atfork.)])
 AC_CHECK_HEADERS(sys/vfs.h, [
-       AC_CHECK_HEADERS(linux/magic.h, [] [AC_MSG_WARN(linux/magic.h is required in order to verify procfs.)])
+       AC_CHECK_HEADERS(linux/magic.h, [], [AC_MSG_WARN(linux/magic.h is required in order to verify procfs.)])
        ], [AC_MSG_WARN(sys/vfs.h is required in order to verify procfs.)])

 AC_ARG_WITH([capability_header],

```

**Comment by stevegrubb** [comment]
Looks like this was fixed in commit 398a149e. I don't think there's been a release since then.

**Comment by svgmuc** [comment]
Indeed. I should have checked HEAD.
